pages/Ore-Washer.md

```diff
@@ -0,0 +1,14 @@
+The Ore Washer is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+
+## Obtaining
+The Ore Washer is a multiblock structure, it consists of a cauldron, a fence and a dispenser.<br>
+**Note:** It doesn't matter which side the dispenser is facing, place it to your liking.<br>
+
+A complete setup should look like this:
+
+![Armor Forge Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-ore-washer.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your washed ores. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Ore Washer can be used to wash ores. Most common use is to wash [Sifted Ore](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Sifted-Ore) into [Dusts](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Dusts).<br>
```

Comment:
"wash ores" sounds misleading, as you cannot place ores into this machine. Maybe remove that part and only say that it is mostly used for sifted ore -> dusts.

Reply:
Sure